string name
uint8 can_id
string param_name
uint32 param_number
---
uint8 ARRAY_PARAMETER = 0
uint8 UINT32_PARAMETER = 1
uint8 INT32_PARAMETER = 2
uint8 FLOAT32_PARAMETER = 3
uint8 UINT16_PARAMETER = 4
uint8 UINT8_PARAMETER = 5

string param_name
uint8 param_type
uint32 value_uint32
uint32 value_int32
float32 value_float32
uint16[2] value_uint16
uint8[4] value_uint8

uint32 ret

